doc/releases/changelog-dev.md

```diff
@@ -41,6 +41,40 @@
 * New basis sets, `6-311g` and `CC-PVDZ`, are added to the qchem basis set repo.
   [#3279](https://github.com/PennyLaneAI/pennylane/pull/3279)
 
+* Support for purity computation is added. The `qml.math.purity` function computes the purity from a state vector or a density matrix:
+
+    ```pycon
+    >>> x = [1, 0, 0, 1] / np.sqrt(2)
+    >>> qml.math.purity(x, [0, 1])
+    1.0
+    >>> qml.math.purity(x, [0])
+    0.5
+    
+    >>> x = [[1 / 2, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 0], [0, 0, 0, 1 / 2]]
+    >>> qml.math.purity(x, [0, 1])
+    0.5
+    ```
+    The `qml.qinfo.purity` can be used to transform a QNode returning a state to a function that returns the purity:
+    ```python3
```

Comment:
```suggestion
    ```

    The `qml.qinfo.purity` can be used to transform a QNode returning a state to a function that returns the purity:

    ```python3
```

Reply:
I am only wondering if RTD isn't parsing this properly, and maybe the newlines would help?

Reply:
This should be resolved by reindentation.